2017-09-20 06:58:13 Error: [MissingActionException] Action TestController::index() could not be found.
Exception Attributes: array (
  'controller' => 'TestController',
  'action' => 'index',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#2 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2017-09-20 06:58:13 Error: Fatal Error (1): Class 'AppHelper' not found in [/media/snadge/Reor/src/cakephp/lib/Cake/View/Helper/HtmlHelper.php, line 33]
2017-09-20 06:58:13 Error: [FatalErrorException] Class 'AppHelper' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 06:58:13 Error: [MissingControllerException] Controller class FaviconController could not be found.
Exception Attributes: array (
  'class' => 'FaviconController',
  'plugin' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 06:58:13 Error: Fatal Error (1): Class 'AppHelper' not found in [/media/snadge/Reor/src/cakephp/lib/Cake/View/Helper/HtmlHelper.php, line 33]
2017-09-20 06:58:13 Error: [FatalErrorException] Class 'AppHelper' not found
Request URL: /favicon.ico
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 06:58:13 Error: [MissingControllerException] Controller class FaviconController could not be found.
Exception Attributes: array (
  'class' => 'FaviconController',
  'plugin' => NULL,
)
Request URL: /favicon.ico
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 06:58:13 Error: Fatal Error (1): Class 'AppHelper' not found in [/media/snadge/Reor/src/cakephp/lib/Cake/View/Helper/HtmlHelper.php, line 33]
2017-09-20 06:58:13 Error: [FatalErrorException] Class 'AppHelper' not found
Request URL: /favicon.ico
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Class 'AppHelpe...', '/media/snadge/R...', 33, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 07:01:48 Error: [MissingActionException] Action TestController::index() could not be found.
Exception Attributes: array (
  'controller' => 'TestController',
  'action' => 'index',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#2 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#3 {main}
2017-09-20 07:01:48 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /css/cake.generic.css
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:01:48 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:01:48 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:03:02 Error: [MissingViewException] View file "Test/json/index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Test/json/index.ctp',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/View/JsonView.php(107): View->_getViewFileName('Test/json/index')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(963): JsonView->render(NULL, NULL)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#4 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2017-09-20 07:03:02 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /css/cake.generic.css
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:03:02 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:03:02 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:03:53 Error: [MissingViewException] View file "Test/json/index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Test/json/index.ctp',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/View/JsonView.php(107): View->_getViewFileName('Test/json/index')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(963): JsonView->render(NULL, NULL)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(200): Controller->render()
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#4 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2017-09-20 07:03:53 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /css/cake.generic.css
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 07:03:53 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 20:15:42 Error: [Error] Class 'HttpSocket' not found
Request URL: /test
Stack Trace:
#0 [internal function]: TestController->test()
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(491): ReflectionMethod->invokeArgs(Object(TestController), Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#4 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2017-09-20 21:28:40 Error: [UnauthorizedException] Unauthorized
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(349): BasicAuthenticate->unauthenticated(Object(CakeRequest), Object(CakeResponse))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(305): AuthComponent->_unauthenticated(Object(TestController))
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#7 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2017-09-20 21:28:40 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(58): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 21:28:40 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(58): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 21:31:10 Error: [UnauthorizedException] Unauthorized
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(349): BasicAuthenticate->unauthenticated(Object(CakeRequest), Object(CakeResponse))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(305): AuthComponent->_unauthenticated(Object(TestController))
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#7 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2017-09-20 21:31:10 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(59): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 21:31:10 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(59): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 21:33:05 Error: [UnauthorizedException] Unauthorized
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(349): BasicAuthenticate->unauthenticated(Object(CakeRequest), Object(CakeResponse))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(305): AuthComponent->_unauthenticated(Object(TestController))
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#7 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2017-09-20 21:33:05 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(59): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 21:33:05 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(59): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(UnauthorizedException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 21:33:54 Error: [UnauthorizedException] Unauthorized
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(349): BasicAuthenticate->unauthenticated(Object(CakeRequest), Object(CakeResponse))
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(305): AuthComponent->_unauthenticated(Object(TestController))
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#7 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2017-09-20 21:34:53 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(697): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(304): AuthComponent->_getUser()
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#9 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2017-09-20 21:35:59 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(697): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(304): AuthComponent->_getUser()
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#9 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2017-09-20 21:36:33 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(697): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(304): AuthComponent->_getUser()
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#9 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2017-09-20 21:36:50 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(697): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(304): AuthComponent->_getUser()
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Utility/ObjectCollection.php(128): AuthComponent->startup(Object(TestController))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#9 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2017-09-20 21:37:41 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users/login
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:38:36 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users/login
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:39:43 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users/login
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:40:32 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users/login
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:40:32 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:40:32 Error: [MissingControllerException] Controller class CssController could not be found.
Exception Attributes: array (
  'class' => 'CssController',
  'plugin' => NULL,
)
Request URL: /css/cake.generic.css
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:40:32 Error: [MissingControllerException] Controller class ImgController could not be found.
Exception Attributes: array (
  'class' => 'ImgController',
  'plugin' => NULL,
)
Request URL: /img/cake.power.gif
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:41:35 Error: [MissingControllerException] Controller class UsersController could not be found.
Exception Attributes: array (
  'class' => 'UsersController',
  'plugin' => NULL,
)
Request URL: /users/login
Stack Trace:
#0 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2017-09-20 21:44:45 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#5 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#10 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2017-09-20 21:44:45 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 21:44:45 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 21:46:48 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#5 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#10 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2017-09-20 21:46:48 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 21:46:48 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(61): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:22:22 Error: [CakeException] Authentication adapter "sessionKey" was not found.
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(768): AuthComponent->constructAuthenticate()
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#2 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#7 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#8 {main}
2017-09-20 22:22:22 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(CakeException))
#6 /media/snadge in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:22:22 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(CakeException))
#6 /media/snadge
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:23:05 Error: [ParseError] syntax error, unexpected '='
Request URL: /test
Stack Trace:
#0 [internal function]: App::load('AppController')
#1 /var/www/html/Controller/TestController.php(3): spl_autoload_call('AppController')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(567): include('/var/www/html/C...')
#3 [internal function]: App::load('TestController')
#4 [internal function]: spl_autoload_call('TestController')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(266): class_exists('TestController')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(235): Dispatcher->_loadController(Object(CakeRequest))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(158): Dispatcher->_getController(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2017-09-20 22:23:05 Error: Fatal Error (16): Exception thrown without a stack frame in [Unknown, line 0]
2017-09-20 22:23:05 Error: [FatalErrorException] Exception thrown without a stack frame
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(16, 'Exception throw...', 'Unknown', 0)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(16, 'Exception throw...', 'Unknown', 0, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:23:26 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#5 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#10 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2017-09-20 22:23:26 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:23:26 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:24:08 Error: [Error] Class 'AppModel' not found
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(125): ClassRegistry::init('User')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#5 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#10 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#11 {main}
2017-09-20 22:24:08 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:24:08 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(Error))
#6 /media/snadge/Reor/sr
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:25:13 Error: [MissingDatasourceConfigException] The datasource configuration "default" was not found in database.php
Exception Attributes: array (
  'config' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/ConnectionManager.php(91): ConnectionManager::_getConnectionObject('default')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3640): ConnectionManager::getDataSource('default')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(1162): Model->setDataSource('default')
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#13 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#14 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#15 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#16 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2017-09-20 22:25:13 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingDatasourceConfigException in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:25:13 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingDatasourceConfigException
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:25:23 Error: [MissingDatasourceConfigException] The datasource configuration "default" was not found in database.php
Exception Attributes: array (
  'config' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/ConnectionManager.php(91): ConnectionManager::_getConnectionObject('default')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3640): ConnectionManager::getDataSource('default')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(1162): Model->setDataSource('default')
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#13 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#14 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#15 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#16 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2017-09-20 22:25:23 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingDatasourceConfigException in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:25:23 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingDatasourceConfigException
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:26:04 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 22:26:04 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:26:04 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:26:11 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 22:26:11 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:26:11 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:38:01 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 22:38:01 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:38:01 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:38:47 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 22:38:47 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:38:47 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 22:39:06 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 22:39:06 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 22:39:06 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:03:12 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 23:03:12 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 23:03:12 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:03:31 Error: [MissingTableException] Table users for model User was not found in datasource default.
Exception Attributes: array (
  'table' => 'users',
  'class' => 'User',
  'ds' => 'default',
)
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3667): Model->setSource('users')
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): Model->getDataSource()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#11 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#13 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}
2017-09-20 23:03:31 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 23:03:31 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(MissingTableException))
#6 /medi
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:05:11 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'User.demo' in 'where clause'
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(472): PDOStatement->execute(Array)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(438): DboSource->_execute('SELECT `User`.`...', Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(675): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(1186): DboSource->fetchAll('SELECT `User`.`...', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): DboSource->read(Object(AppModel), Array)
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#13 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#14 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#15 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#16 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2017-09-20 23:05:11 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(PDOException))
#6 /media/snadge/ in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 23:05:11 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(63): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(PDOException))
#6 /media/snadge/
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:33:41 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'User.demo' in 'where clause'
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(472): PDOStatement->execute(Array)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(438): DboSource->_execute('SELECT `User`.`...', Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(675): DboSource->execute('SELECT `User`.`...', Array, Array)
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Datasource/DboSource.php(1186): DboSource->fetchAll('SELECT `User`.`...', Array)
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3038): DboSource->read(Object(AppModel), Array)
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Model/Model.php(3010): Model->_readDataSource('first', Array)
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BaseAuthenticate.php(129): Model->find('first', Array)
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(97): BaseAuthenticate->_findUser('demo', 'pwd1234')
#8 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/Auth/BasicAuthenticate.php(75): BasicAuthenticate->getUser(Object(CakeRequest))
#9 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(771): BasicAuthenticate->authenticate(Object(CakeRequest), Object(CakeResponse))
#10 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(Object(CakeRequest), Object(CakeResponse))
#11 /var/www/html/Controller/AppController.php(64): AuthComponent->login()
#12 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#13 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#14 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#15 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TestController), Object(CakeRequest))
#16 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#17 {main}
2017-09-20 23:33:41 Error: Fatal Error (1): Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(64): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(PDOException))
#6 /media/snadge/ in [/media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php, line 766]
2017-09-20 23:33:41 Error: [FatalErrorException] Uncaught TypeError: Argument 1 passed to AuthComponent::identify() must be an instance of CakeRequest, null given, called in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php on line 611 and defined in /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php:766
Stack trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Component/AuthComponent.php(611): AuthComponent->identify(NULL, NULL)
#1 /var/www/html/Controller/AppController.php(64): AuthComponent->login()
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Event/CakeEventManager.php(243): AppController->beforeFilter(Object(CakeEvent))
#3 /media/snadge/Reor/src/cakephp/lib/Cake/Controller/Controller.php(677): CakeEventManager->dispatch(Object(CakeEvent))
#4 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(158): Controller->startupProcess()
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ExceptionRenderer.php(95): ExceptionRenderer->_getController(Object(PDOException))
#6 /media/snadge/
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(1, 'Uncaught TypeEr...', '/media/snadge/R...', 766, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:49:45 Error: [ParseError] syntax error, unexpected ')', expecting ']'
Request URL: /test
Stack Trace:
#0 [internal function]: App::load('AppController')
#1 /var/www/html/Controller/TestController.php(3): spl_autoload_call('AppController')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(567): include('/var/www/html/C...')
#3 [internal function]: App::load('TestController')
#4 [internal function]: spl_autoload_call('TestController')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(266): class_exists('TestController')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(235): Dispatcher->_loadController(Object(CakeRequest))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(158): Dispatcher->_getController(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2017-09-20 23:49:45 Error: Fatal Error (16): Exception thrown without a stack frame in [Unknown, line 0]
2017-09-20 23:49:45 Error: [FatalErrorException] Exception thrown without a stack frame
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(16, 'Exception throw...', 'Unknown', 0)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(16, 'Exception throw...', 'Unknown', 0, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
2017-09-20 23:58:02 Error: [ParseError] syntax error, unexpected ')'
Request URL: /test
Stack Trace:
#0 [internal function]: App::load('AppController')
#1 /var/www/html/Controller/TestController.php(3): spl_autoload_call('AppController')
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(567): include('/var/www/html/C...')
#3 [internal function]: App::load('TestController')
#4 [internal function]: spl_autoload_call('TestController')
#5 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(266): class_exists('TestController')
#6 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(235): Dispatcher->_loadController(Object(CakeRequest))
#7 /media/snadge/Reor/src/cakephp/lib/Cake/Routing/Dispatcher.php(158): Dispatcher->_getController(Object(CakeRequest), Object(CakeResponse))
#8 /var/www/html/index.php(107): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
2017-09-20 23:58:02 Error: Fatal Error (16): Exception thrown without a stack frame in [Unknown, line 0]
2017-09-20 23:58:02 Error: [FatalErrorException] Exception thrown without a stack frame
Request URL: /test
Stack Trace:
#0 /media/snadge/Reor/src/cakephp/lib/Cake/Error/ErrorHandler.php(212): ErrorHandler::handleFatalError(16, 'Exception throw...', 'Unknown', 0)
#1 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(970): ErrorHandler::handleError(16, 'Exception throw...', 'Unknown', 0, Array)
#2 /media/snadge/Reor/src/cakephp/lib/Cake/Core/App.php(943): App::_checkFatalError()
#3 [internal function]: App::shutdown()
#4 {main}
